managers/backup-manager/BackupWatcher.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const catalog = require('../../broker/lib/models/catalog');
+const eventmesh = require('../../eventmesh');
+const config = require('../../common/config');
+const logger = require('../../common/logger');
+const CONST = require('../../common/constants');
+const bm = require('./');
+const BaseManager = require('../BaseManager');
+const DBManager = require('../../broker/lib/fabrik/DBManager');
+const errors = require('../../common/errors');
+const Conflict = errors.Conflict;
+
+/* jshint nonew:false */
+new DBManager(); //to start the BnRStatusPoller
+
+class DefaultBackupManager extends BaseManager {
+
+  registerWatcher() {
+    logger.info(`Registering Backup watcher`);
+    return eventmesh.server.registerWatcher('backup', 'defaultbackup', this.worker);
+  }
+
+  worker(change) {
+    function acquireProcessingLock() {
```

Comment:
Can you please put some jsdoc for these functions to detail out what it does?

Reply:
Done!